stl/inc/yvals_core.h

```diff
@@ -325,6 +325,7 @@
 // P2302R4 ranges::contains, ranges::contains_subrange
 // P2321R2 zip
 //     (changes to pair, tuple, and vector<bool>::reference only)
+// P2322R6 ranges::fold

```

Comment:
I thought we write these comments to match the proposal title 🤔

Reply:
Good question - the tracking issues have "cleaned-up" titles, which I personally create when reading through the list of accepted motions. In this case, (as @strega-nil-ms observed), the paper is titled `ranges::fold` but doesn't actually define anything by that name :scream_cat: which I found to be highly confusing. Therefore I cleaned up the title to mention the most prominent things that it actually defines, adding an "etc." because it defines several more things.

Nicole is also correct that our convention is to exactly copy the cleaned-up tracking issue titles into the `yvals_core.h` comments for consistency.

Reply:
Tiny question: should I also capitalize `E` in `etc.`?
Full comment would be `// P2322R6 ranges::fold_left, ranges::fold_right, Etc.`.

Reply:
No, it's not the start of a new sentence. When in doubt, copy the title exactly and if anyone disagrees you can blame STL 😄.

Reply:
> No, it's not the start of a new sentence.
Well, most of the comments have uppercase letters in the middle of the them, that's why I'm asking:

https://github.com/microsoft/STL/blob/main/stl/inc/yvals_core.h#L317-L319

Reply:
Oh right, totally forgot that Title Case is used. I think technically it should be but IMO it looks weird and perhaps STL made an exception here?